lib/tapioca/config_builder.rb

```diff
@@ -10,9 +10,13 @@ class << self
 
       sig { params(command: Symbol, options: T::Hash[String, T.untyped]).returns(Config) }
       def from_options(command, options)
-        Config.from_hash(
-          merge_options(default_options(command), config_options, options)
-        )
+        merged_options = merge_options(default_options(command), config_options, options)
+
+        puts(<<~MSG) if options.include?("generate_command")
```

Comment:
This line now needs to be:
```suggestion
        puts(<<~MSG) if merged_options.include?("generate_command")
```